OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -707,6 +707,61 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 						node.Key = "Refinery";
 				}
 
+				if (engineVersion < 20150310)
+				{
+					// Note: These rules are set up to do approximately the right thing for maps, but
+					// mods need additional manual tweaks.  This is the best we can do without having
+					// much smarter rules parsing, because we currently can't reason about inherited traits.
```

Comment:
This is a single comment that flows over multiple lines.  It does begin with a capital :)